schedule.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title type="text">My Schedule</title>
```

Comment:
Nu e necesar de pus type-ul pe title deoarece html-ul se asteapta ca acesta deja sa fie un text
```suggestion
    <title>My Schedule</title>
```